### DISCUSSION TITLE
Should FromFactory customization on enums ignore the factory?

### DISCUSSION THREAD

**Discussion by Quogu** [AutoFixture/AutoFixture#1373] [Q&A]
I've raised a bug for this: https://github.com/AutoFixture/AutoFixture/issues/1372 as I'm fairly confident this is not intended, but as the issue template requests that I raise it here too, here goes:

Adding a FromFactory customization to a fixture to specify the value of an enum does not use the value from the customization, but instead appears to produce random values of the enum's underlying type. Is this intentional?

Please see the attached issue for demo code.

**Comment by aivascu** [comment]
@Quogu thank you for raising the issue. Please see the reply in #1372.
Even if you are confident that some behavior might be a bug, please confirm in the discussions, before submitting an issue.
